Utilities/CommonDriver.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mars_Luiz.Utilities
+{
+    public class CommonDriver
+    {
+        public static IWebDriver driver;
```

Comment:
please finish implementing this 

public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }